### DISCUSSION TITLE
Missing Standardization Header for PolyDb Component

### DISCUSSION THREAD

**Discussion by ebenezerarunkumar** [nasa/fprime#1899] [Q&A] [answered]
Hello,

I was looking to use the Svc/PolyDb component but I was not able to because it looks like there is a mismatch in the class name(PolyDbImpl vs PolyDb). I think this is because of a missing standardization header? Please let me know if I am mistaken.

I opened up a PR to fix this: #1898.

**Comment by LeStarch** [comment, marked as answer]
This is exactly correct!

**Reply by LeStarch** [reply]
PR approved.

**Reply by ebenezerarunkumar** [reply]
Cool, thank you! One other question - is there any way this commit could be pushed into the master branch? My team is using release v3.1.1 and we'd like this functionality without having to upgrade to a newer release of fprime(whenever a newer one gets released). 

If not, no worries, just wanted to ask.

**Reply by LeStarch** [reply]
I might suggest this:  add this header to your `config` directory:

`<project>/config/Svc/PolyDb/PolyDb.hpp`

That way you can roll with the exact release v3.1.1 and also get that header included. Typically we don't push to master, and even if we did, you'd still not have the fix on release v3.1.1.

**Reply by ebenezerarunkumar** [reply]
That worked, thank you!
